datastream-server/src/main/java/com/linkedin/datastream/server/EventProducer.java

```diff
@@ -271,8 +272,11 @@ private DatastreamRecordMetadata helperSendOrBroadcast(DatastreamProducerRecord
 
     try {
       // Send the event to the transport
-      String destination =
-          record.getDestination().orElse(_datastreamTask.getDatastreamDestination().getConnectionString());
+      String destination = "dummy";
```

Comment:
Is `dummy` destination okay for NOOP Transport provider?

Reply:
It is OK for NoOp TransportProvider (it doesn't lookup the destination string). Changed it to an empty string constant to make it more pleaseant.